lua/entities/cfc_parachute/init.lua

```diff
@@ -129,17 +134,37 @@ function ENT:Think()
         return
     end
 
+    if self._chuteHealth <= 0 then -- so propsurf breaks it
+        self:Close()
+        owner:ChatPrint( "Your chute broke..." ) -- meh print
+        self:EmitSound( "physics/cardboard/cardboard_box_impact_bullet1.wav", 75, 100, 1 )
+        self._chuteNextOpen = CurTime() + CHUTE_BROKEN_DELAY
+        self._chuteHealth = CHUTE_MAX_HEALTH
```

Comment:
Could be its own method in case you want to break it for some other reason

Reply:
Yeah